api4/bot.go

```diff
@@ -89,7 +90,8 @@ func patchBot(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec := c.MakeAuditRecord("patchBot", audit.Fail)
 	defer c.LogAuditRec(auditRec)
-	auditRec.AddMeta("bot_id", botUserId)
+	auditRec.AddEventParameter("bot_id", botUserId)
```

Comment:
```suggestion
	auditRec.AddEventParameter("id", botUserId)
```

Let's use `id` for all entities to make analysis easier. The context helps to understand which id it is.